### DISCUSSION TITLE
Anyone having issues downloading the prisma binaries right now?

### DISCUSSION THREAD

**Discussion by richardryangarcia** [prisma/prisma#19031] [Q&A] [answered]
I'm tryin to run prisma generate to generate a client but for some reason its not working at the moment? I've never had any issues with this before, is this a common occurrence? Thanks for the help! 

prisma and @prisma/client versions are ^4.11.0

![image](https://user-images.githubusercontent.com/31261908/235490388-c403d82a-a291-4b3f-8d24-1cfc32dead5c.png)

**Comment by richardryangarcia** [comment]
I was able to download the requested binary directly from the url in the browser. Is there a way to run generate using the local binary instead of downloading from remote ?

**Comment by richardryangarcia** [comment]
I was using versions 4.13.0 and downgrading to 4.11.0 fixed issue. For anyone else that runs into this

**Reply by nurul3101** [reply]
Hey @richardryangarcia 👋 
That's strange. Do you get the same error on 4.12.0 as well?

**Comment by richardryangarcia** [comment]
just tried it and yes 4.12.0 works. just to be sure i reinstalled 4.13 again (both prisma and @prisma/client) and the issue was still present. downgrading to 4.12.0 fixed the issue and generated the client no problem

**Reply by nurul3101** [reply]
Can you try changing your network and try to generate the client on 4.13.0?

**Reply by nurul3101** [reply]
Let us know if that helps or if you found a solution.

**Comment by jgreywolf** [comment]
I am getting the same/similar error (see below), using 4.12.0. 
If I go to the URL directly, it says "_the specified key does not exist_".
This just started this morning, as I was able to run generate just fine. I checked with other members of my team, and they are not seeing any errors. (the only difference being that I am the only one on Windows).

I only have one network, so not really an option to change the network.

```
Downloading Prisma engines for Node-API for windows [                    ] 0%
Error: request to https://binaries.prisma.sh/all_commits/659ef412370fa3b41cd7bf6e94587c1dfb7f67e7/windows/query_engine.dll.node.gz.sha256 failed, 
reason: read ECONNRESET
```

**Reply by nurul3101** [reply]
I think it should not have to do anything with Prisma version, because for Richard(in above conversation) this error was resolved in 4.12.0 version.
Is there any firewall in your network?

**Reply by janpio** [reply]
This URL works for me on Windows as well.

**Reply by jgreywolf** [reply, marked as answer]
Even after reboot I still had the problem. However, when I opened vscode again, I killed the terminal session and recreated it.

Then everything worked as expected

So not sure I understand at all what was happening

**Reply by janpio** [reply]
Connection problems unfortunately often are like that. Good you could find a way to make it work for you.
